cli/src/main/java/com/box/l10n/mojito/cli/command/checks/AbstractCliChecker.java

```diff
@@ -0,0 +1,57 @@
+package com.box.l10n.mojito.cli.command.checks;
+
+import com.box.l10n.mojito.cli.command.extraction.AssetExtractionDiff;
+import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.okapi.extractor.AssetExtractorTextUnit;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public abstract class AbstractCliChecker {
+
+    protected CliCheckerOptions cliCheckerOptions;
+
+    protected List<AssetExtractionDiff> assetExtractionDiffs;
```

Comment:
Q: what's the rational this being an attribute of this object? Should it be an input of the run methods? You can instantiate a checker of this type and potentially run it on many diffs.

Reply:
This is a hangover from when I was running the commands in parallel and needed to implemented the Callable interface, I'll tidy it up and  pass the list of diffs to the run method